lib/msal-browser/src/request/RequestHelpers.ts

```diff
@@ -71,9 +71,8 @@ export async function initializeBaseRequest(
         );
     }
 
-    // Set requested claims hash if claims-based caching is enabled and claims were requested
+    // Set requested claims hash if claims were requested
     if (
-        config.cache.claimsBasedCachingEnabled &&
```

Comment:
Doesn't this change the behavior here? We are effectively defaulting this option to false so this block would never be hit. Do I have that right?

Reply:
We're removing `claimsBasedCachingEnabled` so this block would be hit as long as `request.claims` is set. In my discussion with @sameerag, the consensus was that claims still need to be able to be set on request for CAE purposes but we never want to cache claims.

Reply:
I guess I'm confused how this works today then when `claimsBasedCachingEnabled` is false? Do we have a bug today?

Reply:
It works the way @jo-arroyo  described above, claims are added to the request but never cached. The bug is if this is `true` and user calls claims per request with a high frequency, the cache starts hitting its limit and requests fail post that point.

Reply:
Let's chat offline, I think we're conflating 2 related but different things and I think this specific change is incorrect.